vms/proposervm/vm_test.go

```diff
@@ -808,11 +809,7 @@ func TestExpiredBuildBlock(t *testing.T) {
 	proVM := New(
 		coreVM,
 		Config{
-			Upgrades: upgrade.Config{
-				ApricotPhase4Time:            time.Time{},
-				ApricotPhase4MinPChainHeight: 0,
-				DurangoTime:                  mockable.MaxTime,
-			},
+			Upgrades:            upgradetest.GetConfigWithUpgradeTime(upgradetest.ApricotPhase4, time.Time{}),
```

Comment:
Does the (Go) zero-value `time.Time{}` have a special meaning?

And the Unix zero-value? Is this effectively the same as using `upgrade.InitiallyActiveTime`?

Reply:
These values seem to be depended on in the tests that use this code... I'm hoping that we can remove the reliance on `time.Time{}` and `time.Unix(0, 0)` as we continue to fix these tests... But it felt out of scope for this PR...

TL;DR: I don't know... Changing them breaks the tests... The tests should be improved in a later PR.